packages/components/scripts/wrapper-generator/VueWrapperGenerator.ts

```diff
@@ -65,7 +65,8 @@ ${[importsFromVue, importsFromUtils, importsFromTypes].filter((x) => x).join('\n
           const defaultPropValue = isDefaultValueComplex ? `() => (${defaultValue})` : defaultValue;
 
           const eslintAnnotation =
-            (component === 'p-headline' || component === 'p-heading') && key === 'color'
+            (component === 'p-headline' || component === 'p-heading' || component === 'p-link-tile-model-signature') &&
+            (key === 'color' || key === 'headingTag')
```

Comment:
Why would `headingTag` be in conflict with any default types?

Reply:
Was needed, cant recap anymore why. I removed it and it seems to work now